expanse.service

```
@@ -0,0 +1,16 @@
+[Unit]
+Description=web app for storing Reddit items
```

```suggestion
Description=fully selfhosted multi-user web app for externally storing Reddit items (saved, created, upvoted, downvoted, hidden) to bypass Reddit's 1000-item listing limits
```
use full description

---

This change I made to keep in mind limited terminal line width when using systemctl, but I have no strong feelings about it if you'd rather keep the long description in the unit file.